syntax = "proto3";

service TextProcessor {
  rpc ProcessText (TextRequest) returns (TextResponse);
}

message TextRequest {
  string text = 1;
}

message TextResponse {
  string processed_text = 1;
}
